csharp/ql/src/semmle/code/csharp/Assignable.qll

```diff
@@ -311,9 +346,9 @@ module AssignableInternal {
      * Holds if the `ref` assignment to `aa` via call `c` is uncertain.
      */
     cached
-    predicate isUncertainRefCall(Call c, AssignableAccess aa) {
-      isRelevantRefCall(c, aa) and
-      exists(ControlFlow::BasicBlock bb, Parameter p | isAnalyzableRefCall(c, aa, p) |
+    predicate isUncertainRefCall(RefArg arg) {
```

Comment:
I think formulating it this way works better because then we need to look for the existence of a path from the target callable entry point to the target callable exit point which does not go through a (certain) assignment to the `ref` parameter.